argus/action_manager/windows.py

```diff
@@ -115,32 +116,76 @@ def get_installation_script(self):
         self.download_resource("windows/installCBinit.ps1",
                                r"C:\installCBinit.ps1")
 
+    def _execute_function(self, cmd, count=util.RETRY_COUNT,
+                          delay=util.RETRY_DELAY, command_type=None):
+        return self._client.run_command_with_retry(
+            cmd, count=count, delay=delay, command_type=command_type
+        )[0]
+
+    def check_cbinit_installation(self, python_path):
+        """Check if Cloudbase-Init was installed successfully."""
+        LOG.info("Checking Cloudbase-Init installation ...")
+
+        check_cmd = r'cd "{}" ; .\python.exe -c "import cloudbaseinit"'.format(
+            python_path)
+
+        try:
+            self._client.run_remote_cmd(
+                cmd=check_cmd, command_type=util.POWERSHELL)
+        except exceptions.ArgusError as exc:
+            LOG.error("Cloudbase-Init installation failed: %s", exc)
+            return False
+
+        LOG.info("Cloudbase-Init was successfully installed!")
+        return True
+
+    def cbinit_cleanup(self, cbinit_path):
+        """Clean up Cloudbase-Init."""
+        LOG.info("Cleaning up Cloudbase-Init ...")
+
+        self.rmdir("{}\\..".format(cbinit_path))
```

Comment:
`os.path.dirname()` is not working because given path is a windows-path and the platform on which Python runs, in our case, is Linux.